flexget/plugins/urlrewrite/newpct.py

```diff
@@ -42,8 +49,22 @@ def parse_download_page(self, url):
         if len(torrent_ids) == 0:
             raise UrlRewritingError('Unable to locate torrent ID from url %s' % url)
         torrent_id = torrent_id_prog.search(torrent_ids[0]).group(1)
-        return 'http://www.newpct.com/descargar/torrent/%s/dummy.html' % torrent_id
+        return 'http://www.newpct.com/torrents/%s.torrent' % ('{:0>6}'.format(torrent_id))
```

Comment:
Couldn't this be rewritten to `'http://www.newpct.com/torrents/{:0>6}.torrent'.format(torrent_id)`?